specification/containerservice/Fleet.Management/update/run.tsp

```diff
@@ -121,6 +121,7 @@ model ManagedClusterUpgradeSpec {
 The upgrade type.
 Full requires the KubernetesVersion property to be set.
 NodeImageOnly requires the KubernetesVersion property not to be set.
+ControlPlaneOnly requires the KubernetesVersion property to be set, and will only be applied to only the ControlPlane and not the AgentPool. (since 2024-02-02-preview)
```

Comment:
typo, you wrote: 
> "and will **only** be applied to **only** the Controlplane"

"and will only be applied to the ControlPlane

Reply:
fixed typo

Reply:
I don't think we should specify since 2024-02-02-preview. please remove it.

no chance that we will be consistent in documenting it this way in the future. this is metadata. if the property is there, good, and you don't care since when. if it's not there because you're on an older version, you don't get the info...

Reply:
reworked the UpgradeSpec description, issue no longer applicable